bliss/encoder.py

```diff
@@ -91,8 +91,8 @@ def __init__(
     @property
     def dist_param_groups(self):
         # create a unique parameter for each per-band flux
-        star_fluxes = [f"star_log_flux {bnd}" for bnd in self.sdss_bands]
-        gal_fluxes = [f"galsim_flux_{bnd}" for bnd in self.sdss_bands]
+        star_fluxes = [f"star_log_flux {bnd}" for bnd in SloanDigitalSkySurvey.BANDS]
+        gal_fluxes = [f"galsim_flux_{bnd}" for bnd in SloanDigitalSkySurvey.BANDS]
```

Comment:
Using a class constant for the bands is a nice change btw :) We should definitely start using this in other places once this gets merged in.